src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
```

Comment:
According to the checklist, using the `*` selector is not recommended as it impacts performance. It's better to set styles only for elements that require them.